contracts/LiquidPledgingBase.sol

```diff
@@ -294,66 +428,90 @@ contract LiquidPledgingBase is Owned {
         PaymentState paid
         ) internal returns (uint64)
     {
-        bytes32 hPledge = sha3(owner, delegationChain, intendedProject, commitTime, oldPledge, paid);
+        bytes32 hPledge = sha3(
+            owner, delegationChain, intendedProject, commitTime, oldPledge, paid);
         uint64 idx = hPledge2idx[hPledge];
         if (idx > 0) return idx;
         idx = uint64(pledges.length);
         hPledge2idx[hPledge] = idx;
-        pledges.push(Pledge(0, owner, delegationChain, intendedProject, commitTime, oldPledge, paid));
+        pledges.push(Pledge(
+            0, owner, delegationChain, intendedProject, commitTime, oldPledge, paid));
         return idx;
     }
 
+    /// @notice A getter to look up a Admin's details
+    /// @param idAdmin The id for the Admin to lookup
+    /// @return The PledgeAdmin struct for the specified Admin
     function findAdmin(uint64 idAdmin) internal returns (PledgeAdmin storage) {
         require(idAdmin < admins.length);
         return admins[idAdmin];
     }
 
+    /// @notice A getter to look up a Pledge's details
+    /// @param idPledge The id for the Pledge to lookup
+    /// @return The PledgeA struct for the specified Pledge
     function findPledge(uint64 idPledge) internal returns (Pledge storage) {
         require(idPledge < pledges.length);
         return pledges[idPledge];
     }
 
-    // a constant for the case that a delegate is requested that is not a delegate in the system
+    // a constant for when a delegate is requested that is not in the system
     uint64 constant  NOTFOUND = 0xFFFFFFFFFFFFFFFF;
 
-    // helper function that searches the delegationChain fro a specific delegate and
-    // level of delegation returns their idx in the delegation chain which reflect their level of authority
-    function getDelegateIdx(Pledge n, uint64 idDelegate) internal returns(uint64) {
+    /// @notice A getter that searches the delegationChain for the level of
+    ///  authority a specific delegate has within a Pledge
+    /// @param n The Pledge that will be searched
+    /// @param idxDelegate The specified delegate that's searched for
+    /// @return How many Delegates have more authority than the specified 
```

Comment:
not quite. it returns the index of the delegate in the delgationChain for the pledge being searched

Reply:
"If the delegate chain contains the delegate with the `admins` array index `idxDelegae` this returns that delegates  corresponding index in the delegationChain. Otherwise it returns  the maximum address."